ibm/data_source_ibm_enterprise_account_groups.go

```diff
@@ -114,25 +115,40 @@ func dataSourceIbmEnterpriseAccountGroups() *schema.Resource {
 	}
 }
 
-func getEnterpriseNext(next *string) (string, error) {
+func getEnterpriseNext(next *string) string {
```

Comment:
Please catch the error url.Parse and return the error also as part of function

Reply:
@hkantare Updated the code as per review